cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -713,3 +716,67 @@ def test_custom_button_open_url_service_obj(objects, button_group):
         1550002
     """
     pass
+
+
+@pytest.fixture(params=["Service", "Other"], scope="module")
+def unassigned_btn_setup(request, appliance, provider, service):
+    if request.param == "Service":
+        btn_type = request.param
+        obj = MyService(appliance, name=service.name)
+        destinations = [ViaUI, ViaSSUI]
+    else:
+        btn_type = "Provider"
```

Comment:
All the other categories(Vm , Instance) will come under Provider ?

Reply:
I don't think we need to cover more objects. we have 28 means 27 other than service. so I just choose one other than service.

Reply:
In that case we can just provide the second parameter as "Provider"